### ISSUE TITLE
ItemPriceContainer.php - A non well formed numeric value encountered in Cart Component

### ISSUE THREAD

**Issue by davidprovaznikj** [oc-shopaholic/oc-orders-shopaholic-plugin#87]
Hello,
I have a problem.
I get exception (A non well formed numeric value encountered in Cart Component) on cart page.

I found the mistake in code: \Lovata\OrdersShopaholic\Classes\PromoMechanism\ItemPriceContainer::getPriceValueAttribute

**I created workaround:**

```
protected function getPriceValueAttribute()
    {
        $fPrice = 0;
        foreach ($this->arUnitPriceList as $fUnitPrice) {
            $numberUnitPrice = floatval(str_replace(',', '.', $fUnitPrice));
            $fPrice += $numberUnitPrice;
        }

        $fPrice = PriceHelper::round($fPrice);

        return $fPrice;
    }
```

Can you repair this on every places in code and create update package?
Thanks.

![image](https://user-images.githubusercontent.com/48086285/71518632-1f258b80-28b4-11ea-8462-10fdbabca4d4.png)

**Comment by kharanenka** [comment]
How can i reproduce bug?

**Comment by davidprovaznikj** [comment]
> How can i reproduce bug?

I'm using Agora theme and products with one offer and multiple price columns:
![image](https://user-images.githubusercontent.com/48086285/71518977-b50de600-28b5-11ea-8d53-d9c6573e7ef6.png)

**Comment by kharanenka** [comment]
Does bug have connection with calculation of shipping price?
Please check where price is passed as string to ItemPriceContainer class. Please check blocks with new ItemPriceContainer()

**Comment by roulendz** [comment]
So to get this BUG. @kharanenka 

1. Start with Price format - Fractional part separator **DOT**.
2. Then, when you have some products, change price seperator, to **comma**, 
3. Clear cache!, 
4. You should get an arror in any page where is `Cart.get()` got the error.

**Comment by kharanenka** [comment]
Fixed in version 1.24.0
